Java/sort/ShellSort.java

```diff
@@ -0,0 +1,52 @@
+class ShellSort 
```

Comment:
Add a beginner pitch about your Code. Take a look at the other Code to better understand what I am looking for. It should be just a brief intro about the Algorithm in multi-line comments.

Reply:
Ohh okay, I am on it.